collector/settings.mdx

```diff
@@ -410,12 +410,28 @@ a self-managed VM on a cloud provider, the configuration settings here may be us
   <tbody>
     <tr>
       <td>db_log_location (<code>LOG_LOCATION</code>)</td>
-      <td>n/a, required for Log Insights</td>
+      <td>[none]</td>
       <td>database log file or directory location (must be readable by pganalyze system user)</td>
     </tr>
+    <tr>
+      <td>db_log_syslog_server (<code>LOG_SYSLOG_SERVER</code>)</td>
+      <td>[none]</td>
+      <td>local address (host:port) to listen on for syslog messages (see below)</td>
+    </tr>
   </tbody>
 </table>
 
+### Syslog-based Log Insights support
+
+If your database logs are available via syslog, you can use the `db_log_syslog_server` setting to have
+the collector listen for syslog messages from a remote server. This should be a local address like
+"0.0.0.0:32514" and it's recommended to use an unprivileged port number (1024 and up) to avoid running
+the collector as root.
+
+Note that you need to use protocol RFC5424, with an unencrypted TCP connection. Due to syslog not being
+an authenticated protocol it is recommended to only use this integration over private networks.
```

Comment:
I'm thinking this might be best as its own dedicated docs page - e.g. at `/docs/log-insights/setup/syslog-server`? (and then link it from here, as well as `/docs/log-insights/setup`)